bluesky/tests/conftest.py

```diff
@@ -2,7 +2,8 @@
 from bluesky.run_engine import RunEngine
 from bluesky.examples import Mover, SynGauss
 import pytest
-
```

Comment:
As with the mds imports, please move these inside the db fixture so they fail at runtime, not import time